server/api/account.js

```diff
@@ -166,16 +181,19 @@ module.exports.init = function(server) {
             registered: new Date(),
             username: req.body.username,
             email: req.body.email,
-            emailHash: crypto.createHash('md5').update(req.body.email).digest('hex'),
+            emailHash: req.body.enableGravatar ? crypto.createHash('md5').update(req.body.email).digest('hex') : DefaultEmailHash,
+            enableGravatar: req.body.enableGravatar,
             verified: false,
             activiationToken: activiationToken,
             activiationTokenExpiry: formattedExpiration,
             registerIp: req.get('x-real-ip')
         };
 
+        await downloadAvatar(newUser);
```

Comment:
In other projects I would have done this download (as well as sending email) in some sort of background job queue. That way, if there's latency between us and Gravatar, the user isn't stuck waiting until it finally clears or it times out.

Perhaps we can achieve something similar here without a job queue. What would the consequences be if we didn't `await` the download? Just fire it off, and forget it?